build.sbt

```diff
@@ -3,55 +3,24 @@ import com.typesafe.tools.mima.core.{ProblemFilters, ReversedMissingMethodProble
 val scala3Version = "3.1.0"
 
 ThisBuild / organization := "org.typelevel"
+ThisBuild / tlBaseVersion := "3.0"
```

Comment:
What is this for? Do we have to bump it after releasing 3.1?

Reply:
It's part of the versioning scheme inherited from sbt-spiewak.
https://github.com/djspiewak/sbt-spiewak/blob/main/versioning.md#compatibility-version

Actually, you should bump it at the time you introduce "3.1-worthy" changes. E.g. if you follow strict semver and only do bugfixes in 3.0.x, your first PR adding a new feature should bump `tlBaseVersion` to 3.1. This is not enforced, but it's good practice and it makes sure all snapshots from that point are versioned with 3.1-hash-SNAPSHOT.

`tlBaseVersion` is also important if you ever decide you want to make breaking changes. At that point you would bump it to 4.0, and this would indicate to MiMa to stop checking bincompat against the 3.x artifacts.

Reply:
Ok, thanks for explaining - I think we should be already on 3.1 (since Scala native forces us to use Scala 3.1) but that can be bumped later as well.